clone/README.md

```diff
@@ -18,40 +34,32 @@ MariaDB [(none)]> GRANT ALL PRIVILEGES ON google_xenon2022.* TO 'clonetool'@loca
 MariaDB [(none)]> FLUSH PRIVILEGES;
 ```
 
-Now you can clone the log with:
-
-```
-go run ./clone/cmd/ctclone --alsologtostderr --v=1 --log_url https://ct.googleapis.com/logs/xenon2022/ --mysql_uri 'clonetool:letmein@tcp(localhost)/google_xenon2022'
-```
+## Tuning
 
-See the optional flags in the `ctclone` tool to configure tuning parameters.
-## Docker
-
-To build a docker image, run the following from the `trillian-examples` root directory:
+The clone library logs information as it runs using `glog`.
+Providing `--alsologtostderr` is provided to any tool using the library, you should see output such as the following during the cloning process:
 
 ```
-docker build . -t ctclone -f ./clone/cmd/ctclone/Dockerfile
+I0824 11:09:23.517796 2881011 clone.go:71] Fetching [4459168, 95054738): Remote leaves: 95054738. Local leaves: 4459168 (0 verified).
+I0824 11:09:28.519257 2881011 clone.go:177] 1202.8 leaves/s, last leaf=4459168 (remaining: 90595569, ETA: 20h55m21s), time working=24.2%
+I0824 11:09:33.518444 2881011 clone.go:177] 1049.6 leaves/s, last leaf=4465183 (remaining: 90589554, ETA: 23h58m29s), time working=23.0%
+I0824 11:09:38.518542 2881011 clone.go:177] 1024.0 leaves/s, last leaf=4470430 (remaining: 90584307, ETA: 24h34m21s), time working=23.3%
 ```
 
-This can be pointed at a local MySQL instance running outside of docker using:
+When tuning, it is recommended to also provide `--v=1` to see more verbose output.
+In particular, this will allow you to see if the tool is encountering errors (such as being told to back off) by the log server if you see lines such as `Retryable error getting data` in the output.
```

Comment:
`by the log server if you see` -> ` e.g. if you see log lines such as...`?

Reply:
Sure.